lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.Model.rb

```diff
@@ -336,9 +336,36 @@ def handle_multi_building_area_types(model, climate_zone, default_hvac_building_
       wateruse_equipment.additionalProperties.setFeature('building_type_for_swh', type_for_swh)
     end
 
+    # ============================Process airloop info ============================================
```

Comment:
As we discussed, move this to a new method, called from handle_user_data(). The method handle_user_data() is a new method which will also call handle_multi_building_area_types(), as well as future user data related processing.

Reply:
Updated: https://github.com/NREL/openstudio-standards/blob/economizer_exception_prm/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb#L655